src/CourseDashboard.cpp

```diff
@@ -82,3 +95,66 @@ void CourseDashboard::updateUser(User & user)
         break;
     }
 }
+
+void CourseDashboard::saveUsersToFile(std::vector<User>  users)
```

Comment:
why do we have a parameter here? users vector is available inside CourseDashboard class.